src/components/App/App.js

```diff
@@ -11,6 +11,9 @@ import '../../stylesheets/Kmedia.css';
 class App extends Component {
   render() {
     const { i18n, store, history, initialI18nStore, initialLanguage } = this.props;
+    if (document.activeElement) {
```

Comment:
@guya This breaks SSR (3001) as there is no `document` on the server.

Moreover, this prevents setting / keeping focus on elements we do want have focus ( input fields like search boxes)

Reply:
The scroll position going the the active item when the page refresh.
If we want to preserve the active item, the scroll is not a bug but the browser functionality.